.github/workflows/pull_request.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: Install apt dependencies
         run: sudo apt-get update -y && sudo apt-get install -y python3-dev openssl libssl-dev gcc pkg-config libffi-dev libxml2-dev libxmlsec1-dev
       - name: Install dependencies
-        run: pip install -r piptools_requirements3.txt && pip install -r requirements3.txt
+        run: pip install wheel && pip install -r piptools_requirements3.txt && pip install -r requirements3.txt
```

Comment:
what is installing wheel for?

Reply:
For some reason it's not already on the host and you can't install wheels without it.

Reply:
This is definitely some quirk of using -e in the requirements file, so I can remove it when we switch back to using the upstream.

Reply:
ty for that context